### PR TITLE
#142: MUI - teams and user settings

### PR DESCRIPTION
**Pull request by anthonybernardi**
teams and user settings

### CODE REVIEW

File: src/frontend/src/pages/HomePage/UpcomingDeadlines.tsx [hunk 1]
```diff
@@ -100,7 +100,7 @@ const UpcomingDeadlines: React.FC = () => {
         </FormControl>
       }
     >
-      <Container fluid>{workPackages.isLoading ? <LoadingIndicator /> : fullDisplay}</Container>
+      <Grid container>{workPackages.isLoading ? <LoadingIndicator /> : fullDisplay}</Grid>
```
Comment (RChandler234) [inline src/frontend/src/pages/HomePage/UpcomingDeadlines.tsx:103]: image

The version with the grid doesn't have the box we have on prod; we probably wanna keep some kind of separators between the work packages

File: src/frontend/src/pages/ChangeRequestDetailPage/ImplementedChangesList.tsx [hunk 2]
```diff
@@ -3,44 +3,39 @@
...
+          <ListItem key={idx}>
+            <Tooltip
+              id="tooltip"
+              title={`${fullNamePipe(ic.implementer)} - ${datePipe(ic.dateImplemented)}`}
+              placement="right"
+            >
+              <Typography>
+                [{<Link href={`${routes.PROJECTS}/${wbsPipe(ic.wbsNum)}`}>{wbsPipe(ic.wbsNum)}</Link>}] {ic.detail}
+              </Typography>
+            </Tooltip>
```
Comment (RChandler234) [inline src/frontend/src/pages/ChangeRequestDetailPage/ImplementedChangesList.tsx:34]: Could you make the tooltip bigger, it's kinda small/ hard to read on my screen

File: src/frontend/src/pages/ChangeRequestDetailPage/ActivationDetails.tsx [hunk 3]
```diff
@@ -3,47 +3,32 @@
...
+        <Grid item xs={12} md={8}>
+          <b>Start Date: </b> {datePipe(cr.startDate)}
+        </Grid>
+        <Grid item xs={12} md={4}>
+          <b>Project Manager: </b> {fullNamePipe(cr.projectManager)}
+        </Grid>
+        <Grid item xs={12} md={8}>
+          <b>Confirm WP Details: </b> {booleanPipe(cr.confirmDetails)}
+        </Grid>
+      </Grid>
```
Comment (RChandler234) [inline src/frontend/src/pages/ChangeRequestDetailPage/ActivationDetails.tsx:31]: image
Prod looks like this with the spaces in between, you can get the same effect by making the actual info in their own grid items. I think it'd be worth making the change bc this looks a bit cramped

**Review by RChandler234 — CHANGES_REQUESTED** [review]

**Comment by anthonybernardi** [comment]
Screen Shot 2022-10-22 at 4 33 21 PM
Screen Shot 2022-10-22 at 4 42 05 PM
Screen Shot 2022-10-22 at 4 42 56 PM

**Pull request merged**